spokestack/context.py

```diff
@@ -77,7 +99,25 @@ def is_active(self, value: bool) -> None:
         Args:
             value (bool): Boolean to set context activity
         """
-        self._active = value
+        self._is_active = value
+
+    @property
+    def is_managed(self) -> bool:
```

Comment:
I haven't read the entirety of this PR, but I was skimming through and noticed this. This property is something I had to shoehorn in to deal with sharing the microphone with Android platform components; it might not be necessary at all in Python.

Reply:
thank you! yeah we decided to remove the managed stuff so I will make sure this is removed also.